docs/index.rst

```diff
@@ -306,6 +306,9 @@ Disks
     (in milliseconds)
   - **busy_time**: (*Linux*, *FreeBSD*) time spent doing actual I/Os (in
     milliseconds)
+  - **time_in_queue**: (*Linux*) total wait time for all requests (in
+    milliseconds)
```

Comment:
In order to be consistent with the established naming convention this field should be called `queue_time`.
Also change the explanation to "number of milliseconds that I/O requests have waited on this block device".